history-of-the-quakers-v2/original/17-book-3-chapter-23.adoc

```diff
@@ -388,7 +408,9 @@ as their trophies, by our pretended forfeitures for conscience-sake.
 William Mead, being left alone at the bar,
 also addressed the jury in the following speech:
 
-"`Ye men of the jury, I now stand here to answer to an indictment against me,
+[quote]
```

Comment:
So I know there is a lot of debate about the [.quote] vs. embedding vs. leaving it as is, but it seemed to me that this should be offset somehow (and it is in the original) and idk how you want to do that, if at all! I asked Jason about this kind of stuff the other day because he was online, and I said maybe a [.embed.document.SPEECH] label would be helpful and he told me to mention that to you. So idk about this one, as it is a verbal quote opposed to a written one, but it's not a document either, which I associate with embedding as well! So just let me know what to do! 😄

Reply:
yeah, haha, there is debate indeed.  jason doesn't seem to like quotes as much as I do maybe, but then again I keep waffling back and forth on whether i like them for various things or not too. 😄 i think let's leave this. it feels weird calling it an embedded document because it's not a document. but i'll keep mulling it over to see if i can clarify our "rules" for embeds and quotes.